syncd/RedisClient.cpp

```diff
@@ -893,17 +893,46 @@ void RedisClient::processFlushEvent(
             SWSS_LOG_THROW("unknown fdb flush entry type: %d", type);
     }
 
-    for (int flush_static: vals)
+    // If has a lot of macs(example:217600) and use lua scripts, will cause REDIS BUSY.
+    // Change to this without atomicity operation.
```

Comment:
Atomic here would be desired sine it could lead to some wired issues while new Mac is learned during deletion